docs/Other/Translating.md

```diff
@@ -123,3 +123,10 @@ A great example is from @SpacedOutChicken's mod Deciv. @The Bucketeer made some
 Your warmongering ways are unacceptable to us. = 即使在野蠻的荒土世界，窮兵黷武的行徑還是無法容忍的！
 (English meaning: These warmongering ways are still unaccepable enen in this world of savage)
 ```
+
+Befor the End
+
+If you are Chinese, congratulations on your good luck!
+Here is a tutorial video (like a nanny) specially prepared for Chinese translators by Unciv Chinese developers.
```

Comment:
Please remove the 'like a nanny'... if we have first-time translators the last thing we want is for them to think that we're looking down on them...

Reply:
OK, I will revise it, but in my Chinese content, according to the Chinese culture, this word means: content like  is very detailed.
I'm sorry I didn't think that he might have ambiguities.